src/main/java/net/sf/mzmine/modules/masslistmethods/chromatogrambuilder/ChromatogramBuilderTask.java

```diff
@@ -171,6 +172,9 @@ public void run() {
 	// Add new peaklist to the project
 	project.addPeakList(newPeakList);
 
+        // Add quality parameters to peaks
+        new QualityParameters(newPeakList);

```

Comment:
Yes, this way looks a bit strange as there is no need for the instance. This is normally done using a static method, e.g. 

public class QualityParameters {
    public static void calculateQualityParams(PeakList peakList) {
     ....
    }
}

and then

QualityParameters.calculateQualityParams(peakList);